src/test/java/uk/gov/companieshouse/scanupondemand/orders/api/ScanUponDemandApiApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.scanupondemand.orders.api;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
Is this annotation required here (yet)?

Reply:
looks like it is not, it was just copied from the template found here: https://companieshouse.atlassian.net/wiki/spaces/DEV/pages/1295614679/CH+QuickStart+Spring+Boot+App+Template